lib/host.py

```diff
@@ -67,7 +67,7 @@ def scp(self, src, dest, check=True, suppress_fingerprint_warnings=True, local_d
             suppress_fingerprint_warnings=suppress_fingerprint_warnings, local_dest=local_dest
         )
 
-    def xe(self, action, args={}, check=True, simple_output=True, minimal=False, force=False):
+    def xe(self, action, args={}, formated_args=[], check=True, simple_output=True, minimal=False, force=False):
```

Comment:
I don't think we need a new parameter here. We could improve `args` so that a value could be a dict.

Example:

```
args = {
    'vif': {
        uuid1: network1,
        uuid2: network2,
        uuid3: network3
    }
}
```
And then format it appropriately for `xe`.

Reply:
Indeed